game/src/main/scala/io/aigar/game/Cell.scala

```diff
@@ -14,7 +15,7 @@ object Cell {
   final val MinMass = 1 // TODO determine this once we have visualization/etc.
 }
 
-class Cell(startPosition: Vector2 = new Vector2(0f, 0f)) {
+class Cell(id: Int, startPosition: Vector2 = new Vector2(0f, 0f)) {
```

Comment:
Is there an advantage to manually manage the ID? Rather than putting some random ones.